demo/README.md

```diff
@@ -0,0 +1,33 @@
+# Demo Application
+
+**Warning**: This is just a simple demo setup that is not suited for production use!
+
+## Prerequisites
+
+1. Get your mashine's IP address
+1. Replace `IP_PLACEHOLDER` with your IP in `.env`.
+1. Run `docker compose up postgres-keycloak keycloak nginx` from `./demo`
+1. Wait until keycloak is ready. Look for "Listening on: http://0.0.0.0:8080." in the docker compose logs.
+1. Go to http://<YOUR_IP>/auth
+1. Login with `admin:admin`
+1. Click on `create realm` in the upper left dropdown
+1. Replace all occurences of `IP_PLACEHOLDER` in `keycloak/init_data/keycloak_masterportal_realm.json` with your mashine's IP address
+1. Paste the content of `keycloak_masterportal_realm.json` into the input field in your browser
+1. Click `create`
+1. Stop the containers (e.g. `ctrl + c`)
+1. Ensure sufficient permissions for `./form-backend/uploads/` (e.g. `chmod -R 777 form-backend/uploads`)
```

Comment:
dir does not exist, maybe add note (create uploads)

Reply:
it is checked in